### DISCUSSION TITLE
Validate Kerberos Ticket against ldap.

### DISCUSSION THREAD

**Discussion by DanielMGoldberg** [dsbenghe/Novell.Directory.Ldap.NETStandard#265] [General]
Hi, I would like to know if there's a way to validate a kerberos ticket received by a client against LDAP using this library.

For example I have a ticket "YL...." and  want to know if its valid so i need to ask the AD. 

Thank you !

**Comment by dsbenghe** [comment]
It is not clear to me what the scenario you want support for, but no, there isn't any support whatsoever for Kerberos tickets in its library.

If you are asking about authentication via Kerberos, there is no support built-in yet see https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/issues/28 but it can be done on top of the library via t extension points.
